guidance.md

```diff
@@ -25,9 +25,9 @@ The request context **SHALL** include:
 ### DRAFT of ServiceRequest Profile
 - status - FHIR value set [RequestStatus](http://build.fhir.org/valueset-request-status.html)
 - intent - FHIR value set [RequestIntent](http://build.fhir.org/valueset-request-intent.html)
-- code - Procedure or imaging study being requested. LOINC? CPT? SNOMED? Both?
+- code - CPT (extensible). Recommend SNOMED/LOINC if CPT is unavailable. Systems should also include Codings with more detail (e.g. via proprietary or local codes as a last resort)
 - subject - Patient
-- reasonCode - ICD?
+- reasonCode - ICD-10 or SNOMED CT (Extensible + Max-ValueSet) This requires SNOMED or ICD10, or free text if your system has neither.  
```

Comment:
Can you run me through the value of "Extensible + max"?  Doesn't "max" mean you can't use other codes? In the call we talked about and Drew advocated for "preferred" here, I think.

Reply:
Preferred doesn't restrict at all (which is ok!)  -- any codes + text are allowed.

[Extensible + max](https://build.fhir.org/ig/HL7/US-Core-R4/general-guidance.html#required-binding-for-codeableconcept-datatype) has been heavily debated on Zulip, and there is still some disagreement about name and purpose (primarily from standards folks), The name isn't great, but here is the definition:

"Required binding to a value set definition means that one of the codes from the specified value set SHALL be used and using only text is not valid. In this IG, we have defined the Extensible + Max-ValueSet binding to allow for either a code from the specified value set or text. Multiple codings (translations) are permitted as is discussed below."